napari/_qt/layer_controls/qt_points_controls.py

```diff
@@ -208,11 +211,15 @@ def __init__(self, layer) -> None:
 
         self.layout().addRow(button_row)
         self.layout().addRow(self.opacityLabel, self.opacitySlider)
-        self.layout().addRow(trans._('point size:'), self.sizeSlider)
+        self.layout().addRow(trans._('current point size:'), self.sizeSlider)
```

Comment:
I think changing the programmatic names helps clarify what's happening in the code. I understand the intent of making the GUI label consistent, but my initial reaction is not great. 

Mostly because they just take up too much valuable horizontal space after this

<img width="311" alt="Screenshot 2023-04-13 at 9 01 04 AM" src="https://user-images.githubusercontent.com/2608297/231818691-4d389ffb-a607-43a5-a3cc-1fde2c9d2d1c.png">

My other issue is with the semantics of what `current` means, so publicizing those semantics more widely (i.e. in the GUI) worries me. One description is that it's the size of the currently selected point, but that's not true since I still see something when no points are selected. Or maybe it's the size of the next of the point I add - but then shouldn't it be `next point size`? Also, changing this when multiple points are selected will actually change the sizes of those points, so maybe this should be just be `point size`.

Also, when I've brought up this complexity before, I've heard that we're reusing the interaction people might be used in word processing software for selecting the font or font size of selected/next text, which seems pretty reasonable. But those software typically don't prefix the label or tooltip with current or selected - it's just `font` or `font size`.

In short, consider just leaving the label alone. But if you feel strongly here, I won't block the review on it.

Reply:
> One description is that it's the size of the currently selected point, but that's not true since I still see something when no points are selected. Or maybe it's the size of the next of the point I add - but then shouldn't it be next point size?

This is an actual conflation in the code, not just semantics; we don't distinguish between these two cases. I do feel that `current point size` more intuitively encapsulates "selected point sizes" and "current default size" than just `point size`, which several times caused people (me included) to expect a different behaviour. The most confusing thing to me is: if you have no points selected and you move the slider, nothing happens. I would expect *all* point sizes to change in that case.

I don't feel strongly about the label, but I think there are more upsides than downsides... though to be fair, I don't typically use small screens, so I'm not affected by the size issue.

Reply:
I'm on Andy's side here — I definitely agree (I've probably made the point myself) that the analogous interaction is wysiwyg UIs for text or shapes, e.g. MS Word or Keynote. (Or LibreOffice if you prefer @brisvag! 😂) In none of those cases is the word "current" used, but they behave the same way as these controls — and if we diverge in behaviour somewhere, we should harmonise with those interactions, rather than move away from them. (For example, when multiple points with different face colors are selected, the face color box should "blank out" in some way, and clicking on it and selecting a color should find the right one.

So anyway, yes, I would ask to revert this change. And I feel more strongly about it than @andy-sweet! 😂 I actually think our controls are too big already!

Reply:
Man has someone used PowerPoint recently? It's hideous! 😂 All of the formatting elements *disappear* when you are in "shape adding" mode, which I find even more confusing. And the *arrangement* when you're editing shape formatting???

<img width="191" alt="Screenshot 2023-04-14 at 10 59 47 pm" src="https://user-images.githubusercontent.com/492549/232050446-40ab7052-9403-4f1d-9d25-edaa22fea0d0.png">

Anyway, not trying *that* again... 😂 

> your cursor is always somewhere and visible, so it's obvious where a change is happening. Here, it can be nowhere (i.e: no selection). You're maybe arguing that in that case it "at the end of the points", but it's not visible or obvious.

I really disagree that this is confusing — and btw in Word you can definitely scroll out of the view of the cursor and still have text formatting options and not think "omg what does this apply to??" It applies to either the *next* thing you type or to the thing that is selected, if any.

Regarding your interpretation of "point size", I would argue that "current point size" could be equally misinterpreted, and @andy-sweet is right that "next point size" makes more sense — unless you have a selection. We could switch the label around as people select and deselect, but I don't think we should go there just yet.

So I would simply add a tooltip here — "choose the size of selected points or points to be added" — and leave the text as is.

> In fact, in those cases where it can be nowhere (e.g: powerpoint when you select the text box, rather than having the cursor inside), those controls do affect the whole content :P

Erm, that's selecting the whole text inside the box, ie cheating 😜. If you truly have nothing selected, then the controls disappear (though they used to control the next thing... I guess someone at Microsoft agrees with you 🤣)

Reply:
> We could switch the label around as people select and deselect, but I don't think we should go there just yet.

Yeah, agree this is not great. Let's just stick with the status quo for now :)